src/main/scala/com/johnsnowlabs/nlp/embeddings/SentenceEmbeddings.scala

```diff
@@ -70,7 +70,11 @@ class SentenceEmbeddings(override val uid: String) extends AnnotatorModel[Senten
         begin = sentence.start,
         end = sentence.end,
         result = sentence.content,
-        metadata = Map.empty[String, String],
```

Comment:
We talked about this with @saif-ellafi since the content would be the entire sentence or worse the document, it is not a good idea to repeat this in metadata and add more memory/storage.
At the moment the SentenceEmbeddings has no annotator to use its metadata so we don’t know what should be the correct metadata. 
Maybe we can leave it empty for the time being?

Reply:
I know it’s coming from other PR, just thought we can discuss it here :)

Reply:
Removing content from metadata requires a larger refactor. I suggest leaving this like it is now and let's work on that as a feature request for a breaking release.

Reply:
Instead of removing it, can we make the content empty? This will really increase the size of the DataFrame and there is no useful point to it.